examples/numalogic-simple-pipeline/ml_steps/README.md

```diff
@@ -0,0 +1,52 @@
+# Simple Numalogic Pipeline 
```

Comment:
let's name the package something like `simpleproj` or `simplepl`. Basically to differentiate between other example src that would be added later